pipeline/transform/traffic_intensity.py

```diff
@@ -14,8 +14,7 @@
 df = pd.read_csv(args.input_file)
 
 # Split the period (e.g. 2015-2019) into start and end year
```

Comment:
Just would remove this comment since for this dataset we found out that there are no period ranges like the population dataset